crates/compiler/gen_dev/src/generic64/mod.rs

```diff
@@ -2336,7 +2378,10 @@ impl<
             // Refactor this and switch to one external match.
             // We also could make loadining indivitual literals much faster
             let element_symbol = match elem {
-                ListLiteralElement::Symbol(sym) => *sym,
+                ListLiteralElement::Symbol(sym) => {
+                    self.load_literal_symbols(&[*sym]);
```

Comment:
nit: don't we have just `load_literal_symbol` which removes the need to wrap in a slice?

Reply:
we don't actually. we have `load_literal` but that skips some checks.